src/HDInsight/HDInsight.Test/UnitTests/HttpTests.cs

```diff
@@ -89,6 +104,7 @@ public void CanGrantHttpAccess()
             commandRuntimeMock.Verify(f => f.WriteObject(connectivitysettings), Times.Once);
         }
 
+        [Obsolete]
```

Comment:
same

Reply:
> same

fixed. Remove test

Reply:
fixed

---
In reply to: [278675439](https://github.com/Azure/azure-powershell/pull/9069#discussion_r278675439) [](ancestors = 278675439)